source/_posts/2017/2017-01-10-jak-psat-zajimave-inzeraty-pro-ajtaky.md

```diff
@@ -0,0 +1,183 @@
+
+### Kamarád taky rád
+
+**Když u vás pracuje někdo šikovný, zmiňte ho v inzerátu**. Lidé půjdou spíše za kamarádem nebo někým v komunitě známým, než jen do firmy.
+
+Sám si pamatuju tyhle dvojice (i když už tam někteří nepracují): 
+
+- Filip Procházka - DámeJídlo, Rohlík 
+- Ondrej Mirtes - Slevomat
+- Patrik Votoček - Shipito
+- Vašek Purchart - Lavito
+
+
+### Řeknete si o feedback - dovnitř i ven
+
+**1. Řekněte si o feedback v práci** 
+
+„Co tě nakonec dovedlo k nám místo ke konkurenci?“
+
+**2. Dejte to přečíst někomu zvenčí**
+
+Vaši zaměstnanci vás přirozeně vidí lépe než někdo, koho neživíte. Lidé zvenčí vám dají upřímnější zpětnou vazbu a spíše řeknou, co potřebují pro navázání vztahu.
+ 
+**3. Které dvě věci se ti nejvíc líbí a které dvě bys zlepšil?**    
+    
+Napište 3 lidem z cílovky. Tak máte šanci se dozvědět, jak působíte navenek &ndash; očima člověka, kterého hledáte.
+
+       
+### Svlékněte se
+
+Ukažte, jak to doopravdy děláte. Důkaz místo slibů.
+
+**Děláte open-source?** Ven s ním. Vím, že Skrz, Jobs, Slevomat, PeckaDesign nebo Shopsys mají balíčky na Githubu. Bohužel, ani u jednoho z inzerátů jsem odkaz nenašel.
+
+Více informací mi může dát jenom náhled do zdrojáků aplikace. **Open-source je velmi podceňovaný hiring nástroj.**
+    
+### Odlište se
+
+Prodejte své odlišnosti. Proto k vám lidi jdou, ne protože stejně jako ostatní děláte v PHP, Symfony, Nette a Doctrine. Taková informace by tam měla být, ale určitě ne na předních místech.
+
+### Příklady přitažlivých odlišností
+
+Znám firmy, které:
+
+- **Každý měsíc mají půldenní školení** - dobrovolné a na přání programátorů.
+
+- Znám taky **„každý měsíc máme půldenní školení“**. To je ale povinné a zadání volí ředitel nebo jiné oddělení. Není větší pain, než být na školení „Jak dělat marketing na Instagramu“, když se chci naučit škálovat výkon přes sharding serverů. 
+
+- **Dávají každému ročně 10 000 Kč na vzdělání** a může si sám vybrat, jak je investuje.
+- **Každý poslední pátek v měsící dávají programátorům možnost refaktorovat** jakoukoliv část kódu chtějí.
+       
+
+### Na velikosti a věku nezáleží
+ 
+Pojmy jako junior a senior mi nedávají smysl. Proč?
+ 
+Představte si, že máme 2 lidi, kteří hledají práci:
+
+A. Senior může být **vývojář s 10 lety praxe**, zná 1 framework, který se učil sám a nechce zkoušet druhý, protože ten jeden mu stačí. 
+
+B. Junior může být **19letý borec** před výškou, který říká, že ještě žádný framework nenapsal, tak asi úplně nebude dobrej. To že umí Elastic, Doctrine, Nette a začíná se Symfony nevidí jako výhodu (true story).
+
+**Koho byste vzali?**
+
+A teď si vemzměte, že do inzerátu napíšte *Senior*. Který z nich se vám přihlásí? Lidé se podceňují a o programátorech to platí desetinásob. Myslete na to.
+
+<img src="/assets/images/posts/2017/job-offers/attitude.jpg" class="thumbnail" alt="Attitude for skill">
+
+Zmínil jsem všechno, co jsem chtěl, tak se teď pojďme podívat na konkrétní inzeráty.
```

Comment:
Za mě ještě nezáleží na škole. Měl jsem na pohovoru kluka z matfyzu, který říkal, jak dělá 8 let weby. Nakonec to byla hrozná bída aka soubor HonzovyFunkce.php v rootu nette projektu). Nebo další měl opět za sebou magistra na čvut, že dělá v symfony. A zjistíš, že to měl akorát jako semestrálku na 3 měsíce a jinak neví nic. Naopak spousta lidí, kteří VŠ neměli, ale učili se nebo už pracovali, byli podstatně lépe. A pak v inzerátu vidíš, že požadují VŠ. Přičemž webařina není vůbec o škole, ale o samostudiu.